cpp/models/abm/README.md

```diff
@@ -1,57 +1,127 @@
 # Agent Based Model
 
-This module models and simulates the epidemic using an agent based model (*ABM*) approach. Unlike the SECIR compartment model that uses an ODE, this model simulates the spread of COVID19 in a population with actual persons (the agents) moving throughout the world and interacting with each other.
+This module models and simulates the epidemic using an agent based model (*ABM*) approach. Unlike the SECIR compartment model that uses an ODE, this model simulates the spread of COVID19 in a population with actual persons (the agents) moving throughout locations in the world and interacting (infecting) with each other.
 
 ## Structure
 
 The model consists of the following major classes:
-1. Person: represents the agents of the model. A person has an assigned infection state (similar to the compartments of the SECIR model) and location. 
-2. Location: represents locations in the world where people meet and interact, e.g. home, school, work, social events.
-3. World: collection of all persons and locations.
-4. Simulation: run the simulation and store results.
+
+1. Person: represents the agents of the model. A person has an ID, i.e. a unique number, an age, a location and a list with their assigned locations, i.e. the locations they visit during the simulation. They can perform tests and wear masks. Every person has lists with past and current infections and vaccinations.
```

Comment:
This slightly confuses me. Should a Person really represent multiple agents, or just one each?
Moreover, saying 'a person represents agents' seems the wrong way around. Shouldn't the agent be a representation of a (group of) person(s)?